app/templates/components/forms/admin/content/social-links-form.hbs

```diff
@@ -1,27 +1,27 @@
-<form class="ui form" {{action 'submit' on='submit'}}>
+<form class="ui form {{if isLoading 'loading'}}" {{action 'submit' on='submit'}} novalidate>
   <div class="field">
     <label>{{t 'Support'}}</label>
-    {{input type='url' name='support'}}
+    {{input type='url' name='support' value=socials.supportUrl}}
   </div>
   <div class="field">
     <label>{{t 'Twitter'}}</label>
-    {{input type='url' name='twitter'}}
+    {{input type='url' name='twitter' value=socials.twitterUrl}}
   </div>
   <div class="field">
     <label>{{t 'Facebook'}}</label>
-    {{input type='url' name='facebook'}}
+    {{input type='url' name='facebook' value=socials.facebookUrl}}
```

Comment:
@geekyd which is why we can do this in the model. 

https://github.com/fossasia/open-event-frontend/blob/development/app/models/event.js#L124